kitsune/sumo/static/sumo/js/main.js

```diff
@@ -5,6 +5,29 @@ window.k = window.k || {};
 (function () {
   k.LAZY_DELAY = 500;  // delay to lazy loading scripts, in ms
   k.STATIC_URL = $('body').data('static-url');
+  k.getPathAsDict = function(propertyNames) {
+    var pathDict = {},
+      pathArray = window.location.pathname.split('/').filter(Boolean);
+
+    propertyNames.map(function(value, index) {
+      pathDict[value] = pathArray[index];
```

Comment:
`.map` is for transforming an array into a new array. Since this is just looping over the array it would be better to use `propertyNames.forEach()`.